models/public/wavernn/description/wavernn.md

```diff
@@ -0,0 +1,135 @@
+# wavernn
+
+## Use Case and High-Level Description
+
+WaveRNN is a model for the text-to-speech task originally trained in PyTorch\*
+then converted to ONNX\* format.
+WaveRNN performs waveform regression from mel-spectrogram.
+For details see [paper](https://arxiv.org/pdf/1703.10135.pdf), [repository](https://github.com/as-ideas/ForwardTacotron).
+
+## ONNX Models
+We provide pretrained models in ONNX format for user convenience.
+
+### Steps to Reproduce PyTorch to ONNX Conversion
+Model is provided in ONNX format, which was obtained by the following steps.
+
+1. Clone the original repository
+```sh
+git clone https://github.com/as-ideas/ForwardTacotron
```

Comment:
The rest of this document references the `fatchord/WaveRNN` repository; why is the `as-ideas/ForwardTacotron` repository used here?

Reply:
Because ForwardTacotron was forked from fatchord/WaveRNN and I tested ONNX conversion in this repository.